### ISSUE TITLE
Comments on Thymio_ITA.pdf - "Istruzioni d’uso Thymio"

### ISSUE THREAD

**Issue by Prevo01** [USI-Showroom/Thymio#1]
- **Question**
Before starting should I upgrade the robot with the new firmware as exlained in section 7?

- **Sections 3.1, 3.2, 3.3 and 3.4** 
There is always the instruction: "Scegliere un robot dalla lista..."
At least by using Linux Ubuntu, to choose the robot it is possible to either double click on the visualized robot, or click on the robot icon and then click on the botton below "Program with ..."

- **Section 3.5**
"incluso con il software scaricato al punto 1." should be "incluso il software scaricato al punto 1."
I tried to launch the simulator but it doesn't work.

- **Section 4.1**
The link https://github.com/USI-Showroom/thymio/examples is not working...
The right link is https://github.com/USI-Showroom/thymio/tree/master/examples
I downloaded the files and tried to open them by means of Aseba Studio and Blockly but it didn't work.
Maybe we can discuss this via videoconference during the next days.

- **Section 5**
Is the function available under Scratch?
I started to test the example to but I experienced some problems with the "if-do logic". It seems that it is not possible to remove the "true" condition and add the one proposed by the example and a second problem is related to the fact that it seems not to be possible to add the "else" condition.

In the next days I will go ahead with section 6 and add a new issue if needed.

**Comment by Steeven9** [comment]
> Before starting should I upgrade the robot with the new firmware as explained in section 7?

I think it would be a good idea to update it as soon as possible. Moreover, with Thymio Suite it's really intuitive (although I never tried it, since the ones I have already have the last version installed).

> Is the function available under Scratch?

I don't think so, I cannot find any IR blocks in Scratch

**Comment by Prevo01** [comment]
Today I tried again launching the simulator and it works. Probably yesterday I had too many open applications on my computer...

**Comment by Steeven9** [comment]
> Today I tried again launching the simulator and it works. Probably yesterday I had too many open applications on my computer...

It might be due to a port conflict, I had some problems with the old Aseba software (port 3000 I think) but I didn't test it with Thymio Suite yet.

**Comment by Steeven9** [comment]
> I downloaded the files and tried to open them by means of Aseba Studio and Blockly but it didn't work.

Weird, I tried just now and it works both with a Thymio and a simulator. There was an error in ledTest that has been fixed in d97c8f5 but it shouldn't prevent you from opening the file.
